pages/pipelines/command_step.md

```diff
@@ -306,6 +306,24 @@ _Optional Attributes_
       <em>Example:</em> <code>2</code>
     </td>
   </tr>
+  <tr>
+    <td><code>signal</code></td>
+    <td>
+      The signal sent to a process, if the process did not intercept and handle the signal<br>
+      <em>Example:</em> <code>"*"</code><br>
+      <em>Example:</em> <code>kill</code><br>
+      <em>Example:</em> <code>SIGINT</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>signal_reason</code></td>
+    <td>
+      The reason a process was signalled<br>
```

Comment:
Style changes:

- End the sentence with a period.
- Use US English

```suggestion
      The reason a process was signaled.<br>
```

Reply:
Would you mind adding a period to line 304 in this change, too?

Reply:
Done.